src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.EventGrid</PackageId>
 		<Description>Provides developers with a library to create and manage all Azure EventGrid resources. </Description>
-		<Version>1.1.0-preview</Version>
+		<Version>1.2.0-preview</Version>
```

Comment:
@kalyanaj update package release notes for this release in .csproj

Reply:
Done, thanks.